.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+tmp/
```

Comment:
Is this necessary? It'd be best to remove any temp directories created during tests if possible.